src/main/java/ch/wisv/areafiftylan/products/controller/OrderRestController.java

```diff
@@ -268,6 +269,21 @@ public Order getOrderById(@PathVariable Long orderId) {
         return createResponseEntity(HttpStatus.OK, "Order status updated", order);
     }
 
+    /**
+     * This call allows for assigning a giveaway ticket. It creates the order directly.
+     *
+     * @param userId UserID of the Giveaway ticket
+     * @return The created giveaway Order.
+     */
+    @PreAuthorize("hasRole('ADMIN')")
+    @PostMapping("/{userId}/{ticketType}/assigngiveaway")
```

Comment:
If you want to stay closer to the REST principles, consider moving these path variables to a DTO model as requestbody